src/CustomTargets.fsx

```diff
@@ -13,4 +13,10 @@ Target "Dotnet Restore" (fun _ ->
     DotNetCli.Restore(fun p ->
```

Comment:
This file can be deleted now Jack's changes have been merged

Reply:
done